include/zephyr/settings/settings.h

```diff
@@ -45,6 +45,10 @@ extern "C" {
  */
 #define SETTINGS_EXTRA_LEN ((SETTINGS_MAX_DIR_DEPTH - 1) + 2)
 
+#ifndef CONFIG_SETTINGS_COMMIT_PRIO_MINIMUM
```

Comment:
Is it required? I'd prefer to see compilation error report if it has not been set correctly over KConfig.

Reply:
Normally this would not be required. However the PR kept failing CI, it seems that some parts of the bluetooth subsystem (I think it is gatt) are using the `SETTINGS_STATIC_HANDLER(...)` without enabling settings (and thus not setting `CONFIG_SETTINGS_COMMIT_PRIO_MINIMUM`).